lib/uffizzi/cli/login.rb

```diff
@@ -60,10 +63,73 @@ def handle_succeed_response(response, server, username)
       ConfigFile.write_option(:username, username)
       ConfigFile.write_option(:cookie, response[:headers])
       ConfigFile.write_option(:account_id, account[:id])
+
+      default_project = ConfigFile.read_option(:project)
+      return unless default_project
+
+      check_default_project(default_project, server)
     end
 
     def account_valid?(account)
       account[:state] == 'active'
     end
+
+    def check_default_project(default_project, server)
+      check_project_response = fetch_projects(server)
+      return ResponseHelper.handle_failed_response(check_project_response) unless ResponseHelper.ok?(check_project_response)
+
+      projects = check_project_response[:body][:projects]
+      slugs = projects.map { |project| project[:slug] }
+      return if slugs.include?(default_project)
+
+      question = "Project '#{default_project}' does not exist. Select one of the following projects or create a new project:"
+      choices = projects.map do |project|
+        { name: project[:name], value: project[:slug] }
+      end
+      all_choices = choices + [{ name: 'Create a new project', value: nil }]
+      answer = Uffizzi.prompt.select(question, all_choices)
+      return ConfigFile.write_option(:project, answer) if answer
+
+      create_new_project(server)
+    end
+
+    def create_new_project(server)
+      project_name = Uffizzi.prompt.ask('Project name: ', required: true)
+      generated_slug = Uffizzi::ProjectHelper.generate_slug(project_name)
+      project_slug = Uffizzi.prompt.ask('Project slug: ', default: generated_slug)
+      raise Uffizzi::Error.new('Slug must not content spaces or special characters') unless project_slug.match?(/^[a-zA-Z0-9\-_]+\Z/i)
+
+      project_description = Uffizzi.prompt.ask('Project desciption: ')
+
+      params = {
+        project: {
+          name: project_name.strip,
+          slug: project_slug,
+          description: project_description,
+        },
+      }
+
+      response = create_project(server, params)
+
+      if ResponseHelper.created?(response)
+        handle_create_project_succeess(response)
+      else
+        name_error = response[:body][:errors][:name]
+        name_already_exists = name_error && name_error.first == 'Name already exists'
+        message = "Project with name #{project_name} already exists. " \
+        'Please run $ uffizzi config to set it as a default project'
+        return Uffizzi.ui.say(message) if name_already_exists
```

Comment:
Do we need to raise error here? If so let's create a separate method as with success case